packages/design-picker/package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "@automattic/design-picker",
-	"version": "1.0.0",
+	"version": "1.0.1",
```

Comment:
I don't think this is necessary, and I don't think we bump the version on changes that aren't going to be published. Unlike `calypso-build` and `data-store` packages, `design-picker` isn't published to npm.